CTY-5_builtarea.ipynb

```diff
@@ -1,33 +1,19 @@
 {
```

Comment:
<p>I removed <code>worldpop = ee.ImageCollection("WorldPop/GP/100m/pop").filter(ee.Filter.equals('year', 2020))</p><p>popcodes = worldpop.aggregate_array('country').getInfo()</code> because it looked like it wasn't being used.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wri/SCL-cities/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>